### PR TITLE
created Projects commponent and ProjectsTile component

### LINKED ISSUE #19: Create Project Tile Component

### PR DESCRIPTION
**Pull request by fmtabbara**
closes #19

### CODE REVIEW

File: client/src/app/components/ProjectTile/ProjectTile.js [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+const ProjectTile = ({
+  projectTitle,
+  projectDescription,
+  projectLevel,
+  projectLanguages,
+  projectImage,
+  projectMembers
+}) => {
+  return (
+    <>
```
Comment (manu-4216) [inline client/src/app/components/ProjectTile/ProjectTile.js]: I think you don't need another wrapper <>. You already have the div bellow it.

File: client/src/app/components/Projects/Projects.js [hunk 2]
```diff
@@ -0,0 +1,70 @@
+import React from "react";
+
+import "./projects.css";
+import ProjectTile from "../ProjectTile/ProjectTile";
+
+const projects = [
+  {
+    projectTitle: "Coding for Fun",
```
Comment (manu-4216) [inline client/src/app/components/Projects/Projects.js]: I'd say we don't need the project prefix

File: client/src/app/components/Projects/Projects.js [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+        ({
+          projectTitle,
+          projectDescription,
+          projectLevel,
+          projectLanguages,
+          projectImage,
+          projectMembers
+        }) => {
+          return (
+            <ProjectTile
```
Comment (manu-4216) [inline client/src/app/components/Projects/Projects.js:50]: Same, having project prefix each time is available

File: client/src/app/components/Projects/Projects.js (hunk 2 above)
Comment (manu-4216) [reply client/src/app/components/Projects/Projects.js]: It should match the structure of the projects model in mongoose

File: client/src/app/components/Projects/Projects.js (hunk 3 above)
Comment (manu-4216) [reply client/src/app/components/Projects/Projects.js:50]: *avoidable

File: client/src/app/components/ProjectTile/ProjectTile.js (hunk 1 above)
Comment (fmtabbara) [reply client/src/app/components/ProjectTile/ProjectTile.js]: True. Updated.

File: client/src/app/components/Projects/Projects.js (hunk 2 above)
Comment (fmtabbara) [reply client/src/app/components/Projects/Projects.js]: Updated

File: client/src/app/components/Projects/Projects.js (hunk 3 above)
Comment (fmtabbara) [reply client/src/app/components/Projects/Projects.js:50]: Updated

**Comment by fmtabbara** [comment]
I updated the title to use css but kept the function for description due to the multi-line issues.

**Review by manu-4216 — APPROVED** [review]

**Pull request merged**